src/Raven.Client/Data/SubscriptionConnectionOptions.cs

```diff
@@ -48,6 +50,7 @@ public enum ConnectionStatus
         public MessageType Type { get; set; }
         public ConnectionStatus Status { get; set; }
         public RavenJObject Data { get; set; }
+        public string FreeText { get; set; }
```

Comment:
We use a property named Exception with Type = Error for this.
And we can use that for the disconnect message as well